### PR TITLE
docs: add release dates to v1220 release notes

### PR DESCRIPTION
**Pull request by coreyarnold**
docs: add release dates to v1220 release notes
leave copy/paste date as estimated until we're ready to ship it

### CODE REVIEW

**Review by homelessbirds — DISMISSED** [review]

**Comment by homelessbirds** [comment]
Looks good! thanks :)

File: src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220.mdx [hunk 1]
```diff
@@ -3,9 +3,10 @@ subject: Browser agent
 releaseDate: '2022-10-07'
 version: '1220'
 ---
-<Callout variant="caution" title="About this release">
-This release is in public preview. We're slowly rolling out these features to different browser agents based on the browser's instrumentation method. Full support will be announced soon! 
-    </Callout>
+
+* Internal NR Platform release date: 10/07/2022
```
Comment (a-sassman) [inline src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220.mdx:7]:
```suggestion
<Callout variant="caution" title="About this release">
This release is in public preview. We're slowly rolling out these features to different browser agents based on the browser's instrumentation method. Here's the release timeline:
* Internal NR Platform release date: 10/07/2022
* Production APM-injected release date: 10/7/2022
* Production Standalone release date: Estimated ~12/13/2022
    </Callout>
```

File: src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220.mdx [hunk 2]
```diff
@@ -4,8 +4,13 @@ releaseDate: '2022-10-07'
 version: '1220'
 ---
 
+<Callout variant="caution" title="About this release">
```
Comment (a-sassman) [inline src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220.mdx:7]: @coreyarnold Is it okay to keep it in the caution callout as a preface to the bullet points?

**Comment by homelessbirds** [comment]
I can't get this to build for some reason and eng is out for the day. I will merge first thing Monday morning. Sorry for the delay!

**Review by ghost — APPROVED** [review]

**Pull request merged**